src/types/State.ts

```diff
@@ -62,7 +62,7 @@ export interface Calculator {
  * If you change the schema here without taking precautions, you **will** break existing shortlinks.
  */
 export interface ImportableData {
-  loadouts: Player[];
+  loadouts: Partial<Player>[];
```

Comment:
`PartialDeep<Player>`?

Reply:
How about [SetOptional](https://github.com/sindresorhus/type-fest/blob/HEAD/source/set-optional.d.ts)<Player, 'name'>?

Reply:
I'd rather not have to update the struct every time we change the schema. We're already not using all the fields from Player when we import, so PartialDeep is more fitting.